python/py-requests/Portfile

```diff
@@ -35,13 +35,23 @@ if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools
 
-    depends_lib-append  port:py${python.version}-chardet \
+    depends_lib-append  port:py${python.version}-charset-normalizer \
                         port:py${python.version}-idna \
                         port:py${python.version}-urllib3 \
                         port:py${python.version}-certifi
 
-    if {${python.version} >= 30} {
-        depends_lib-append  port:py${python.version}-charset-normalizer
+    if {${python.version} <= 35} {
```

Comment:
it appears that the `py35` subport was already built for version 2.26.0 at least according to the [buildbots](https://ports.macports.org/all_builds/?port_name=py35-requests). So if this doesn't work at runtime anymore and you want to downgrade to an earlier version you will have to increase (in this case add) an `epoch` field. Otherwise the change will never be picked up for users that have already upgraded the port.

Reply:
Thanks @reneeotten for reviewing this. I added it on line 49 below. This is since py27 is still supported, it's only py35 that requires the epoch.

Reply:
sorry - I missed that one...